### DISCUSSION TITLE
Lightbox

### DISCUSSION THREAD

**Discussion by edenramoneda** [saadeghi/daisyui#2301] [Ideas / feature requests]
It would be great if the lightbox feature could be added to DaisyUI

 [https://tailwind-elements.com/docs/standard/components/lightbox/](https://tailwind-elements.com/docs/standard/components/lightbox/)

**Comment by saadeghi** [comment]
There are many JS plugins for that on NPM.  
It's not possible with CSS only and daisyUI is CSS.
